<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 36, 0);
border-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="show_definitions_button">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>30</y>
     <width>300</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 255);
font-size:15pt; 
color:rgb(243, 243, 243);
selection-background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>show definitions</string>
   </property>
  </widget>
  <widget class="QLabel" name="word_line">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>30</y>
     <width>371</width>
     <height>55</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 255);
font-size:20pt; 
color:rgb(243, 243, 243);
</string>
   </property>
   <property name="text">
    <string>WORD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show_examples_button">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>300</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 255);
font-size:15pt; 
color:rgb(243, 243, 243);
selection-background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>show examples</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="definitions_text">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>120</y>
     <width>300</width>
     <height>550</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
font-size:14pt; color:rgb(243, 243, 243);
selection-background-color: rgb(255, 255, 255);
selection-color: rgb(46, 194, 126);
</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="examples_text">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>300</width>
     <height>550</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 255, 255);
font-size:14pt; color:rgb(243, 243, 243);
selection-background-color: rgb(255, 255, 255);
selection-color: rgb(46, 194, 126);
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shuffle_button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>720</y>
     <width>141</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 255);
font-size:15pt; 
color:rgb(243, 243, 243);
selection-background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>shuffle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_button">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>200</y>
     <width>200</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 255);
font-size:15pt; 
color:rgb(243, 243, 243);
selection-background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pronunciation_UK">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>110</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>WindowsGraphics</normalon>
     <activeon>:/voiceButtonIcon.png/voiceButtonIcon.png</activeon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="pronunciation_UK_text">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 15pt;</string>
   </property>
   <property name="text">
    <string>UK</string>
   </property>
  </widget>
  <widget class="QLabel" name="pronunciation_US_text">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>120</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 15pt;</string>
   </property>
   <property name="text">
    <string>US</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pronunciation_US">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>110</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>WindowsGraphics</normalon>
     <activeon>:/voiceButtonIcon.png/voiceButtonIcon.png</activeon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="button_e">
   <property name="geometry">
    <rect>
     <x>921</x>
     <y>690</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(3, 54, 22);
font: 16pt;</string>
   </property>
   <property name="text">
    <string>e</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="button_d">
   <property name="geometry">
    <rect>
     <x>921</x>
     <y>740</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(3, 54, 22);
font: 15pt;</string>
   </property>
   <property name="text">
    <string>d</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="button_s">
   <property name="geometry">
    <rect>
     <x>921</x>
     <y>640</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(3, 54, 22);
font: 15pt;</string>
   </property>
   <property name="text">
    <string>s</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="button_enter">
   <property name="geometry">
    <rect>
     <x>921</x>
     <y>580</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(3, 54, 22);
font: 16pt;</string>
   </property>
   <property name="text">
    <string>space</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="play_sound_line">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>640</y>
     <width>121</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 255);
font-size:15pt; 
color:rgb(243, 243, 243);
</string>
   </property>
   <property name="text">
    <string>play sound</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="show_examples_line">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>690</y>
     <width>171</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 255);
font-size:15pt; 
color:rgb(243, 243, 243);
</string>
   </property>
   <property name="text">
    <string>show examples</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="show_definitions_line">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>740</y>
     <width>191</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 255);
font-size:15pt; 
color:rgb(243, 243, 243);
</string>
   </property>
   <property name="text">
    <string>show definitions</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="next_line">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>580</y>
     <width>61</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 255);
font-size:15pt; 
color:rgb(243, 243, 243);
</string>
   </property>
   <property name="text">
    <string>next</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_button">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>720</y>
     <width>91</width>
     <height>55</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 255);
font-size:15pt; 
color:rgb(243, 243, 243);
selection-background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
